src/controllers/dynakube/oneagent/daemonset/volumes.go

```diff
@@ -33,7 +35,7 @@ func getCertificateMount() corev1.VolumeMount {
 func getTLSMount() corev1.VolumeMount {
 	return corev1.VolumeMount{
 		Name:      "tls",
-		MountPath: OneAgentCustomKeysPath,
+		MountPath: filepath.Join("/mnt/root", OneAgentCustomKeysPath),
```

Comment:
isn't it `/mnt/host_root`?

Reply:
if I understand correctly we mount `/mnt/root` to `mnt/host_root`, but I want to test it to be sure :)

Reply:
seems to work :D 
```
[root@gke-marcell--test-default-pool-67b1a6d3-m5k1 /]# cat mnt/host_root/var/lib/dynatrace/oneagent/agent/customkeys/custom.pem 
asd
```

Reply:
my "secret": `kubectl -n dynatrace create secret generic tls-secret --from-literal="server.crt=asd"`